javascript/builtins/intl/DateTimeFormat.json

```diff
@@ -163,6 +163,56 @@
                 }
               }
             },
+            "dayPeriod": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "92"
+                  },
+                  "chrome_android": {
+                    "version_added": "92"
+                  },
+                  "edge": {
+                    "version_added": "92"
+                  },
+                  "firefox": {
+                    "version_added": "90"
+                  },
+                  "firefox_android": {
+                    "version_added": "90"
+                  },
+                  "ie": {
+                    "version_added": false
+                  },
+                  "nodejs": {
+                    "version_added": "13.2.0"
```

Comment:
@hamishwillee thanks for the clarification! I tested it (relatively quickly, thanks to nodenv) and it doesn't work in Node 13, 16, or nightly—my guess is that this won't be picked up by Node until Node upgrades to V8 9.2 (Node's nightly builds are at 9.1, right now).

```suggestion
                    "version_added": false
```

Reply:
Thanks, agreed. Done!